### PR TITLE
Added RTL Style for Classic Theme

### LINKED ISSUE #17245: FO in Arabic should be RTL
**Describe the bug**
The FO should be in RTL when in Arabic (and other RTL language too).

**To Reproduce**
1. On BO, install Algeria localization pack
2. Go on FO, change language to Arabic
3. Display is still Left To Right instead of Right To Left

It was OK on 1760, 1761, 1762.

**Screenshots**
In 1.7.6.2 
image (5)

In 1.7.6.3
image (6)

**Additional information**
PrestaShop version: 1.7.6.3 build 1
PHP version: N/A

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.6.x
| Description?  | Added RTL Style for Classic Theme
| Type?         | bug fix
| Category?     | FO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Fixes #17245
| How to test?  | Cf issue

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/17248)

### CODE REVIEW

File: themes/classic/assets/css/error_rtl.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
+#body {
```
Comment (matthieu-rolland) [inline themes/classic/assets/css/error_rtl.css:1]: I assume this file is generated and the weird indentation is normal ?

**Review by PierreRambaud — DISMISSED** [review]
RTL shouldn't be commited :thinking: 
ping @eternoendless

File: themes/classic/assets/css/error_rtl.css (hunk 1 above)
Comment (Progi1984) [reply themes/classic/assets/css/error_rtl.css:1]: Absolutely, it was generated from this : https://devdocs.prestashop.com/1.7/themes/reference/rtl/#stylesheet-generation

**Comment by matks** [comment]
> RTL shouldn't be commited 🤔
> ping @eternoendless

RTL BO assets should not be committed 😄
RTL FO Classic Theme assets ought to be, they were commited for 1.7.6.2, see https://github.com/PrestaShop/PrestaShop/tree/1.7.6.2/themes/classic/assets/css

**Comment by Progi1984** [comment]
> RTL shouldn't be commited thinking
> ping @eternoendless

Actually, no script generate *_rtl file. You must need to generate them from the backoffice. For the next version, an issue #17247 has been created by @matks for automating that.

**Review by matks — DISMISSED** [review]

**Review by PierreRambaud — DISMISSED** [review]

**Comment by Robin-Fischer-PS** [comment]
Hi @Progi1984 ,

I tested BO and FO RTL display in Arabic and Persian, with theses languages as default, and with English as default. I also tested a fresh install in Persian (because Arabic is not possible due to another bug).

All my tests are OK, so it's QA :heavy_check_mark: !

**Review by eternoendless — CHANGES_REQUESTED** [review]
These assets must be generated automatically when installing the language and MUST NOT be committed unless you have a very good reason.

**Review by eternoendless — APPROVED** [review]
Ok go for merging this but we need to make a decision for 1.7.7

**Comment by eternoendless** [comment]
Thank you @Progi1984

**Comment by matks** [comment]
> Ok go for merging this but we need to make a decision for 1.7.7

And develop ! 😄

**Pull request merged**